### PR TITLE
Added UpdateCustemerHander

### PR DESCRIPTION
**Pull request by janbj**

### CODE REVIEW

File: customer-commons/src/test/java/no/unit/nva/customer/service/impl/DynamoDBCustomerServiceTest.java [hunk 1]
```diff
@@ -108,6 +109,18 @@ public void updateExistingCustomerPreservesCreatedDate() throws Exception {
...
 
+    @Test
+    public void updateExistingCustomerWithDifferentIdentifiersThrowsException() throws Exception {
+        Customer customer = getNewCustomer();
+        Customer createdCustomer = service.createCustomer(customer);
+        UUID differentIdentifier = UUID.randomUUID();
+
+        InputException exception = assertThrows(InputException.class,
+            () -> service.updateCustomer(differentIdentifier, createdCustomer));
+        assertEquals(String.format(DynamoDBCustomerService.IDENTIFIERS_NOT_EQUAL,
```
Comment (axthosarouris) [inline customer-commons/src/test/java/no/unit/nva/customer/service/impl/DynamoDBCustomerServiceTest.java:120]:
```
String expectedMessage = String.format(DynamoDBCustomerService.IDENTIFIERS_NOT_EQUAL,differentIdentifier, customer.getIdentifier())
```

Comment (axthosarouris) [inline customer-commons/src/test/java/no/unit/nva/customer/service/impl/DynamoDBCustomerServiceTest.java:120]:
```suggestion
  String expectedMessage = String.format(DynamoDBCustomerService.IDENTIFIERS_NOT_EQUAL,differentIdentifier, customer.getIdentifier());
  assertEquals(expectedMessage, exception.getMessage());
```

File: create-customer/src/test/java/no/unit/nva/customer/create/CreateCustomerHandlerTest.java [hunk 2]
```diff
@@ -53,7 +53,6 @@ public void setUp() {
     }
 
     @Test
-    @DisplayName("Request to Handler Returns Customer Created")
```
Comment (axthosarouris) [inline create-customer/src/test/java/no/unit/nva/customer/create/CreateCustomerHandlerTest.java:56]: No need to fix: Aren't display names easier to read?

File: update-customer/src/test/java/no/unit/nva/customer/update/UpdateCustomerHandlerTest.java [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+                        .with(REQUEST_ID, null)
+                        .build(),
+                getErrorResponseHeaders(),
+                SC_BAD_REQUEST
+        );
+
+        assertEquals(expected, actual);
+    }
+
+    protected InputStream createRequest(String identifier, Object body, Map<String,Object> headers)
```
Comment (axthosarouris) [inline update-customer/src/test/java/no/unit/nva/customer/update/UpdateCustomerHandlerTest.java:120]: this is something that we need in many places during testing: 
https://github.com/BIBSYSDEV/nva-testutils/pull/3

File: create-customer/src/test/java/no/unit/nva/customer/create/CreateCustomerHandlerTest.java (hunk 2 above)
Comment (janbj) [reply create-customer/src/test/java/no/unit/nva/customer/create/CreateCustomerHandlerTest.java:56]: For me it's no difference really and I think it's more work to keep descriptions and test names in sync.

File: update-customer/src/test/java/no/unit/nva/customer/update/UpdateCustomerHandlerTest.java (hunk 3 above)
Comment (janbj) [reply update-customer/src/test/java/no/unit/nva/customer/update/UpdateCustomerHandlerTest.java:120]: Good idea, but you need to extend with request parameters.

**Pull request merged**